.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -449,40 +439,31 @@ jobs:
       uses: seanmiddleditch/gha-setup-vsdevenv@v4
 
     # Ninja is required on Windows for CMAKE_CXX_COMPILER_LAUNCHER to work for using sccache.
-    #
-    # Until debug and release packages are split we only build release packages for vcpkg
-    # to improve CI run times, hence VCPKG_HOST_TRIPLET and VCPKG_TARGET_TRIPLET
-    # https://github.com/microsoft/vcpkg/issues/1626
     - name: "Configure"
       run: >-
         cmake
         -D USE_QT5=${{ matrix.qt_version == 5 }}
         -D CMAKE_BUILD_TYPE=Release
-        -D VCPKG=${{ matrix.vcpkg }}
-        -D VCPKG_HOST_TRIPLET=${{ matrix.vcpkg_triplet }}
-        -D VCPKG_BUILDTREES_ROOT=${{ matrix.vcpkg_buildtrees_root  }}
-        -D VCPKG_TARGET_TRIPLET=${{ matrix.vcpkg_triplet }}
         -G Ninja
         -S . -B build
-      working-directory: ${{ matrix.workspace }}
```

Comment:
Why has this been removed? Doesn't this still need to run in the workspace?

Reply:
Before we had to set a custom workspace as we needed to build vcpkg in a custom location otherwise we would run out of disk space. Now we can use the default folder where everything is normally checked out (note how we don't need the custom checkout task anymore) to and all tasks are by default executed in. (note that the WASM runner is already doing this).

So this was a workaround we had before for vcpkg.

Reply:
Ah, nevermind, we don't need to specify the workspace explicitly anymore without vcpkg, because we're not running out of space in the default directory.
Makes sense.